src/utils/httpClient.ts

```diff
@@ -0,0 +1,8 @@
+const BASE_URL =
+  'https://mate-academy.github.io/react_dynamic-list-of-todos/api/';
+
+export function GetData(url: string) {
+  return fetch(BASE_URL + url).then(response => {
+    return response.json();
+  });
```

Comment:
The `fetch` call lacks error handling. Consider checking `response.ok` and throwing an error if the response is not successful. This will help in managing failed fetch requests gracefully.